analysis/import/SCMLogParser/src/main/java/de/maibornwolff/codecharta/importer/scmlogparser/parser/git/GitLogNumstatParserStrategy.java

```diff
@@ -0,0 +1,94 @@
+package de.maibornwolff.codecharta.importer.scmlogparser.parser.git;
+
+import de.maibornwolff.codecharta.importer.scmlogparser.parser.LogLineCollector;
+import de.maibornwolff.codecharta.importer.scmlogparser.parser.LogParserStrategy;
+import de.maibornwolff.codecharta.model.input.Modification;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class GitLogNumstatParserStrategy implements LogParserStrategy {
+
+    public static final String CORRESPONDING_LOG_CREATION_CMD = "git log --numstat --no-renames";
+    private static final Predicate<String> GIT_COMMIT_SEPARATOR_TEST = logLine -> logLine.startsWith("commit");
+    private static final String AUTHOR_ROW_INDICATOR = "Author: ";
+    private static final String DATE_ROW_INDICATOR = "Date: ";
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("EEE MMM d HH:mm:ss yyyy ZZZ", Locale.US);
+    private static final String FILE_LINE_REGEX = "\\d+\\s+\\d+\\s+\\S+";
+
+    @Override
+    public List<String> listSupportedMetrics() {
+        return Arrays.asList(
+                "code_churn",
```

Comment:
These names are used in other classes, too. Maybe it is better to extract them into a common enumeration. May also be overengineered in this use case but could prevent errors from typo's in the future